### DISCUSSION TITLE
Area of application doesn't work properly

### DISCUSSION THREAD

**Discussion by jonathanlsn** [Open-Systems-Pharmacology/Forum#802] [Questions & Problems] [answered]

**Comment by jonathanlsn** [comment]
Hello,

I'm working on a model about a molecule penetration through the skin with Mobi and I must choose different application area to compare the total penetration of this molecule. But when I change this parameter from 5000 cm^2 to 10000 cm^2 I obtain almost the same results. So I made a test with just 5 cm^2 and I obtained almost the same result again.

I would like to know if you have ever met this problem before ?

Best regards,
Jonathan

**Comment by msevestre** [comment]
Maybe @abdullahhamadeh can help?

**Comment by abdullahhamadeh** [comment]
Hi @jonathanlsn, are you simulating Q? Because that result is normalized by area.  ie if you double the area you get the same Q since it is in dimension mass per area.

**Comment by jonathanlsn** [comment]
Sorry I didn't specified the type of output I want to simulate. Actually I am simulating the Amount of molecule into the plasmatic compartment at a given time point (80h for example). For each value of application area I obtained approximately 70 micromol of my compound.

**Comment by abdullahhamadeh** [comment]
Ok, can you verify what amount gets into in_vivo_sink?  Does it change when you change the area?

**Comment by jonathanlsn** [comment]
There is just one parameter Volume = 1.00 L and it's the same whatever the area.

**Comment by abdullahhamadeh** [comment]
In which compartment? in_vivo_sink?

**Comment by jonathanlsn** [comment]
Yes, the Volume is the only parameter present in in_vivo_sink and its formula type is Constant.

**Comment by abdullahhamadeh** [comment]
Yes, that is ok, because those parameters are not used.  But is the simulated amount of the permeant reaching the in_vivo_sink changing when you modify the area?

**Comment by jonathanlsn** [comment]
Yes, the permeant amount is changing  with the application area but not much. I checked the permeant amount at the same time point for different skin area and I made this graph below. As expected the amount increase linearly with the area but the predicted intercept is equal to 69.01 instead of 0.

![image](https://user-images.githubusercontent.com/47557630/115907202-db6f7180-a468-11eb-93c4-2454cf26b5df.png)

**Comment by abdullahhamadeh** [comment]
Ok please upload your model.  I will look at it and your earlier issue, #684

**Comment by jonathanlsn** [comment]
[Skin area.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/6373415/Skin.area.zip)

Here are two models with 5 and 5000 cm^2 skin area.
Thank you for taking a look on my project.

**Comment by abdullahhamadeh** [comment]
@jonathanlsn the reason you are not seeing a difference when you change the area is because the amount of avobenzone entering via the dermal route is much much smaller than the amount that is entering via the 4x/day application, so you can't see the effect of the dermal application on the graph.  To see this, remove the 4x/day and then plot plasma concentration using different areas of dermal application.  You will find that, eg, if you double the area then you double the amount in plasma.

**Comment by jonathanlsn** [comment]
Ok, but I would like to keep 4x/day application to simulate an in vivo protocol, so the solution would be to set the dose entering via dermal route as the sum of 4x/day application dose ?

**Comment by abdullahhamadeh** [comment]
Do you mean you want to simulate a 4x/day dermal application protocol?  Because right now it seems it is an oral protocol 4x/day.

**Comment by jonathanlsn** [comment]
Could you please show what makes you think that it's an oral protocol ? Because I have never used an oral one, it has always been a 4x/day dermal application. Concerning the application dose in DERMAL_APPLICATION_AREA|skin_compartment|Dose I made sure it was equal to each application dose in 4x/day administration protocol I created on PK-sim.
Thank you for your help.

**Comment by abdullahhamadeh** [comment]
You should use the latest version of the model if you want multiple dermal applications.  The version you are currently using is out of date and can only do one application at time zero.  You need to add the dermal applications in MoBi as shown in the documentation in section 4.2, you can't add them in PKSim.  

I have made an events building block that has dermal applications at the same times as in your original application.   [You can download it from here.](https://github.com/Open-Systems-Pharmacology/Forum/files/6379810/avobenzone_4x_daily_dermal_applications.zip)
Use the current model in the repository, but replace the events building block with the one in the zip file.  You can set the dose as before, from DERMAL_APPLICATION_AREA|skin_compartment|Dose.

**Comment by jonathanlsn** [comment]
Thank you for sharing this building block. I tried it on my project and I finally obtained the Avobenzone amounts in SC, ED, and DE but the evolution of the quantity of drug into the plasma doesn't look like the individual received 4x/day dermal application as you can see on the screenshot. I have the feeling that the individual receive just one administration at 10h. I tried to understand where I can modify this parameter in events building block but i didn't succeed.

![image](https://user-images.githubusercontent.com/47557630/116268264-72943c00-a77d-11eb-9b80-55a7e170ad39.png)

Here is the project with the building block you shared with me.
[test new building block.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/6385729/test.new.building.block.zip)

Please can you take a look on my project again so I can understand where does this problem comes from  ?

**Comment by abdullahhamadeh** [comment, marked as answer]
@jonathanlsn this is because the avobenzone dose is very large and remains on the skin since very little of it is absorbed.   When you add the new dose, the skin is still saturated with avobenzone from the previous application, so you don't see the difference in the absorption profile.

Try it with a dose of 0.01 ug/cm2 and plot the mass of the avobenzone in the vehicle (ie select vehicle_mass when choosing the outputs to plot).  With the small dose, the skin is not saturated and you will be able to see the different applications in the absorption profile (Q.)
